File: backend/result-service/src/main/java/resultService/dto/LoadResult.java
package resultService.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.util.Set;

@AllArgsConstructor
@Setter
@Getter
public class LoadResult {
    private final long key;
    private final long concurrentUsers;
    private final long averageResponseTime;
    private final long maxResponseTime;
    private final String state;
    private final Set<String> errors;
}

File: backend/result-service/src/main/kotlin/resultService/dto/LoadResult.kt
package resultService.dto

class LoadResult(
    var key: Long = 0,
    var concurrentUsers: Int = 0,
    var averageResponseTime: Long = 0,
    var maxResponseTime: Long = 0,
    var state: String? = null,
    var errors: Set<String>? = null,
)
{
}